app/helper/blacklist/helper_test.go

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2022 LimeChain Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package blacklist
+
+import (
+	"github.com/limechain/hedera-eth-bridge-validator/app/clients/hedera/mirror-node/model/transaction"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func Test_IsBlacklistedAccount(t *testing.T) {
+	blacklist := []string{"0x000000", "0x000001"}
+	assert.True(t, IsBlacklistedAccount(blacklist, "0x000000"))
+	assert.False(t, IsBlacklistedAccount(blacklist, "0x000002"))
+}
+
+func Test_CheckTxForBlacklistedAccounts(t *testing.T) {
```

Comment:
I'd split this test into multiple ones, testing each of the for loops and their happy and not happy paths. I know it's a bit tedious but that way we are sure there won't be any sdk changes that would alter the functionality... and we get our coverage up ;) And for the splitting I'd say it's easier to pinpoint which check is failing when there is one per test